extensions/blocks/slideshow/transforms.js

```diff
@@ -57,12 +58,7 @@ const transforms = {
 		{
 			type: 'block',
 			blocks: [ 'core/gallery' ],
-			transform: ( { images } ) => createBlock( 'core/gallery', { images } ),
-		},
-		{
-			type: 'block',
-			blocks: [ 'jetpack/tiled-gallery' ],
-			transform: ( { images } ) => createBlock( 'jetpack/tiled-gallery', { images }, [] ),
```

Comment:
Here I removed "from slideshow to tiled-gallery"  transform in slideshow block, since tiled gallery block already has the same transform:

https://github.com/Automattic/jetpack/blob/22ab2e1ef84599d432db2f3b755fce8cb31b0071/extensions/blocks/tiled-gallery/index.js#L169-L185